### DISCUSSION TITLE
Wrong Vite version in README.md - There is no Vite v.5.0.0

### DISCUSSION THREAD

**Discussion by philipp-serfling** [vitest-dev/vitest#4681] [Q&A] [answered]
I think there is a typoe in the [`README.md`](https://github.com/vitest-dev/vitest#readme)

> Vitest 1.0 requires Vite >=v5.0.0 and Node >=v18.00

Did you mean Vite >=v4.0.0? 
Newest version of Vite ist [v4.4.12](https://github.com/vitejs/vite/releases/tag/v4.4.12)

Thx 🙏

**Comment by sheremet-va** [comment, marked as answer]
Newest version of Vite is 5.0.5 https://github.com/vitejs/vite/releases/tag/v5.0.5

**Comment by philipp-serfling** [comment]
https://github.com/vitejs/vite
huh, so the site is misleading
thx!

<img width="287" alt="image" src="https://github.com/vitest-dev/vitest/assets/463137/687b84ac-437e-48ef-b263-eac6e7c3083a">
